fahari/common/models/common_models.py

```diff
@@ -317,7 +317,7 @@ class RegionType(models.TextChoices):
         WARD = "ward"
 
     user = models.OneToOneField(User, on_delete=models.PROTECT)
-    allotment_type = models.CharField(max_length=10, choices=AllotmentType.choices)
+    allotment_type = models.CharField(max_length=100, choices=AllotmentType.choices)
```

Comment:
Also why change this? 10 characters were enough to contain all the choices of that field.